bitnami/clickhouse/Chart.yaml

```
@@ -32,4 +32,4 @@ maintainers:
 name: clickhouse
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/clickhouse
-version: 9.0.0
+version: 9.0.1
```

Please bump minor version instead given you're adding a new parameter:
```suggestion
version: 9.1.0
```